oca_dependencies

```diff
@@ -0,0 +1,2 @@
+server-tools
```

Comment:
Should the file be named oca_dependencies.txt?

Reply:
Yes :man_facepalming: